### PR TITLE
factory-monitor: buid fix - remove excessive IN and OUT from OIC ports

### PR DESCRIPTION
**Pull request by edersondisouza**
A change in Soletta removed IN_ and OUT_ prefix from OIC ports.

Signed-off-by: Ederson de Souza ederson.desouza@intel.com

### CODE REVIEW

File: factory-monitor/node_types/custom-node.c [hunk 1]
```diff
@@ -301,11 +301,11 @@ custom_pool_new_type(const struct sol_flow_node_type **current)
...
+        { 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__OUT__SET_FAILURE, 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__IN__FAILURE },
 
-        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__OUT_FAILURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_FAILURE },
-        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__OUT_TEMPERATURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_TEMPERATURE },
-        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__OUT_NAME, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_NAME },
+        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__FAILURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_FAILURE },
+        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__TEMPERATURE, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_TEMPERATURE },
+        { 1, SOL_FLOW_NODE_TYPE_MONITOR_CLIENT_TEMPERATURE__OUT__NAME, 0, SOL_FLOW_NODE_TYPE_CUSTOM_NODE_MONITOR_POOL_CONTROLLER__IN__IN_NAME },
```
Comment (glima) [inline factory-monitor/node_types/custom-node.c:308]: Ack. Can you remove the INs/OUTs from your json's (custom node) as well? Like `IN_TEMPERATURE`...

Comment (edersondisouza) [inline factory-monitor/node_types/custom-node.c:308]: May I do that on a different commit? I didn't want to mix build fix with other stuff...

Comment (glima) [inline factory-monitor/node_types/custom-node.c:308]: Sure

**Comment by glima** [comment]
Bump.

**Comment by edersondisouza** [comment]
v2 on #10
